cirq/work/observable_measurement_data.py

```diff
@@ -423,13 +453,28 @@ def variance(self, setting: InitObsSetting, *, atol: float = 1e-8):
             raise ValueError("No measurements")
         self._validate_setting(setting, what='variance')
 
-        _, var = _stats_from_measurements(
+        mean, var = _stats_from_measurements(
             bitstrings=self.bitstrings,
             qubit_to_index=self._qubit_to_index,
             observable=setting.observable,
             atol=atol,
         )
 
+        if self._readout_calibration is not None:
+            ro_setting = _setting_to_z_observable(setting)
```

Comment:
nit: move this line a bit below, closer to the first use of `ro_setting`

Reply:
done